Moves access check to a config

AustinW opened the pull request:
Also adds .gitignore, updates composer.json, refactor routes, add tests and update SlashLoginServiceProvider.

AustinW committed:
Moves access check to a config. Adds .gitignore, update composer.json, refactor routes, add tests and update SlashLoginServiceProvider

AustinW committed:
Add GitHub action to run tests on push and pull request events

FahadYousafMahar reviewed (commented):
Superb work 💯

I've reviewed the code and I really love the way you've added value to this package by refactoring code and adding tests. 

I'll merge pull request once I've ran them.

AustinW commented:
Appreciate that! Yeah I wanted to use the package, but had a need to customize the access check because we have multiple environments to lock down. Love the easy functionality though so thank you for developing this!

FahadYousafMahar committed:
v1.0.2

- Contributions from @AustinW merged.
- Vendor:publish tag renamed in README.md
- Package version Upgraded
